tools/sggolangcilint/tools.go

```diff
@@ -33,16 +33,27 @@ func defaultConfigPath() string {
 }
 
 func CommandInDirectory(ctx context.Context, directory string, args ...string) *exec.Cmd {
-	configPath := filepath.Join(directory, ".golangci.yml")
-	if _, err := os.Lstat(configPath); errors.Is(err, os.ErrNotExist) {
-		configPath = defaultConfigPath()
+	cmdArgs := []string{"run"}
+	var skipConfig bool
+	for _, arg := range args {
+		if arg == "--no-config" {
+			skipConfig = true
+			break
+		}
+	}
+	if !skipConfig {
+		configPath := filepath.Join(directory, ".golangci.yml")
+		if _, err := os.Lstat(configPath); errors.Is(err, os.ErrNotExist) {
+			configPath = defaultConfigPath()
+		}
+		cmdArgs = append(cmdArgs, "-c", configPath)
+	} else {
+		cmdArgs = append(cmdArgs, "--no-config")
 	}
-	var excludeArg []string
 	if directory == sg.FromSageDir() {
-		excludeArg = append(excludeArg, "--exclude", "(is a global variable|is unused)")
+		cmdArgs = append(cmdArgs, "--exclude", "(is a global variable|is unused)")
 	}
-	cmdArgs := append([]string{"run", "-c", configPath}, args...)
-	cmd := Command(ctx, append(cmdArgs, excludeArg...)...)
+	cmd := Command(ctx, append(cmdArgs, args...)...)
```

Comment:
@odsod notice how I changed the way that `--exclude` was earlier placed _after_ the user-supplied arguments (`args`), but I'm changing that here. Do you think that's a problem?

```diff
- cmd := Command(ctx, append(cmdArgs, excludeArg...)...)
+ cmd := Command(ctx, append(cmdArgs, args...)...)
```

Reply:
Shouldn't be a problem, but let's test in your services first?